inductiva/templates/protein_solvation/gromacs/simulation.mdp.jinja

```diff
@@ -4,11 +4,12 @@ nsteps                  = {{ nsteps }}       ; number of steps in simulation
 dt                      = 0.002              ; 2 fs
 
 ; Output control
-nstxtcout               = 1                   ; generate .xtc file 
-nstxout                 = 500                ; save coordinates every 500 iter
+nstxout                 = 500                ; save coordinates every 1.0 ps
```

Comment:
This should be configurable. We usually call this `output_time_step` (to make a distinction with the simulation `time_step`, which should also be configurable). In this case, should it be called `output_time_step_ps` to abide by our previous discussion on parameter units?

Reply:
yes, I was thinking the same! I will expose it to the user then.